graylog2-server/src/test/java/org/graylog/testing/elasticsearch/ElasticsearchInstance.java

```diff
@@ -97,13 +99,16 @@ private static ElasticsearchContainer createContainer(String image, Version vers
     }
 
     private static ElasticsearchContainer startNewContainerInstance(String image) {
+        final Stopwatch sw = Stopwatch.createStarted();
+
         final ElasticsearchContainer container = new ElasticsearchContainer(image)
+                .withReuse(true)
                 .withEnv("ES_JAVA_OPTS", "-Xms512m -Xmx512m")
                 .withEnv("discovery.type", "single-node")
                 .withEnv("action.auto_create_index", "false")
                 .waitingFor(Wait.forHttp("/").forPort(9200));
         container.start();
-        LOG.debug("Started container {}{}", container.getContainerInfo().getId(), container.getContainerInfo().getName());
+        LOG.debug("Started container {} in {}ms", container.getContainerInfo().getName(), sw.elapsed(TimeUnit.MILLISECONDS));
```

Comment:
FYI there is a similar log line printed by the container itself inside `GenericContainer#start`